Server/src/tests/controllers/AdminManagementController.spec.ts

```diff
@@ -8,7 +8,14 @@ describe('Fetch All Categories Materials Controller ', () => {
     let controller: AdminManagementController;
 
     beforeEach(async () => {
-        await createConnection();
+        try {
+            await createConnection();

```

Comment:
isnt there createOrGetConnection?

Reply:
I do not believe so; looking at the source file of getConnectionManager, getConnection, and createConnection there does not appear to be such a function.